### ISSUE TITLE
Question about camera parameters in order to render in Pytorch3d

### ISSUE THREAD

**Issue by JinnanC** [nkolot/SPIN#113]
Hi, thanks for inspiring work!
I have several questions about the representation in SPIN.
1. Is all of the vertices are in the world coordinate?
2. If I want to render the meshes using pytorch3d, how should I convert the currently estimated camera_translation (It seems R is always identity matrix) in the SPIN model to the R,T for the pytorch3d perspective camera? I am confused about the pred_camera.  Here is the description for pytorch3d camera:https://pytorch3d.readthedocs.io/en/latest/modules/renderer/cameras.html#
3. Why the K is set to 5000? It seems the camera to human distance (Z in estimated camera_translation)is around 50-80 if I am correct, and all the vertices are from -1~1, so would this focal length zoom out the original human?

**Comment by acse-ll2819** [comment]
I have the same question as point 3.

**Comment by madsemildahlgaard** [comment]
@JinnanC did you figure out answers to your questions?

**Comment by ahnHeejune** [comment]
in demo.py  
camera_translation = torch.stack([pred_camera[:,1], pred_camera[:,2], 2*constants.FOCAL_LENGTH/(constants.IMG_RES * pred_camera[:,0] +1e-9)],dim=-1)

Can you explain why tz is 

2*constants.FOCAL_LENGTH
-------------------------------------------
constants.IMG_RES * pred_camera[:,0]

=  2 * f  /  Resolution * scale_factor
